templates/citizen_action/list_participants.html.twig

```diff
@@ -17,30 +26,29 @@
-                    {% endif %}
-                </tr>
-                {% for participant in participants %}
-                    <tr>
-                        <td class="member-first-name">{{ participant.firstName|capitalize }}</td>
-                        {% if is_host %}
-                            <td class="member-age">{{ participant.age }}</td>
-                            <td class="member-postal-code">{{ participant.postalCode }}</td>
-                            <td class="member-city-name">{{ participant.cityName }}</td>
-                            <td class="member-subscription-date">
-                                {{- participant.createdAt|date('d/m/Y') -}}
-                            </td>
-                        {% endif %}
-                    </tr>
-                {% endfor %}
-            </table>
+            {% if is_host %}
+                <div class="committee__members__nav">
+                    <div>
+                        <form method="post" action="{{ path("app_citizen_action_contact_participants", { slug: citizen_action.slug, project_slug: citizen_action.citizenProject.slug }) }}">
+                            <input type="hidden" name="token" value="{{ csrf_token('citizen_action.contact_participants') }}" id="members-contact-token" />
+                            <input type="hidden" name="contacts" value="" id="members-contact-selection" />
+                            <button id="members-contact-button" class="btn b__nudge--right" disabled="disabled">Contacter</button>
+                        </form>
+                        <form method="post" action="{{ path("app_citizen_action_export_participants", { slug: citizen_action.slug, project_slug: citizen_action.citizenProject.slug }) }}">
+                            <input type="hidden" name="token" value="{{ csrf_token('citizen_action.export_participants') }}" id="members-export-token" />
+                            <input type="hidden" name="exports" value="" id="members-export-selection" />
+                            <button id="members-export-button" class="btn b__nudge--right" disabled="disabled">Exporter</button>
+                        </form>
+                        <form method="post" action="{{ path("app_citizen_action_print_participants", { slug: citizen_action.slug, project_slug: citizen_action.citizenProject.slug }) }}">
+                            <input type="hidden" name="token" value="{{ csrf_token('citizen_action.print_participants') }}" id="members-print-token" />
+                            <input type="hidden" name="prints" value="" id="members-print-selection" />
+                            <button id="members-print-button" class="btn" disabled="disabled">Imprimer</button>
+                        </form>
+                    </div>
+                </div>
+            {% endif %}
+            
+            {% include 'citizen_action/_participants_list.html.twig' with { participants: participants, is_host: is_host } only %}
```

Comment:
Included templates have access to the variables of the active context, not need `with`